src/components/auth/DownloadDesktopToast.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useMemo } from 'react'
+import { isElectron } from '../../integration/desktop'
+import { isWindows } from '../../integration/browser'
+import './DownloadDesktopToast.css'
+
+export const DownloadDesktopToast:React.FC<React.HTMLAttributes<HTMLDivElement>> = () => {
+  const desktopAvailable = useMemo(() => !isElectron() && isWindows(), [])
+  return (
+    <div style={{ visibility: desktopAvailable ? 'visible' : 'hidden' }}>
+      <a
+        className="DownloadDesktopApp"
+        href="https://decentraland.org/download/"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        Want to play on windows? <span style={{ textDecoration: 'underline' }}>Download the desktop client</span>
```

Comment:
Would you mind creating a class for this component and define it in DownloadDesktopToast?